examples/kitchen-sink.yaml

```diff
@@ -426,3 +426,42 @@ resource:
         - process.command_args
   # Configure the resource schema URL.
   schema_url: https://opentelemetry.io/schemas/1.16.0
+# Configure instrumentation
+instrumentation:
+  # General SemConv options that may apply to multiple languages
+  general:
+    peer:
+      service-mapping:
+        - peer: 1.2.3.4
+          service: FooService
+        - peer: 2.3.4.5
+          service: BarService
+    http:
+      client:
+        request:
+          capture-headers:
+            - X-Foo
+        response:
+          capture-headers:
+            - X-Foo
+      server:
+        request:
+          capture-headers:
+            - X-Bar
+        response:
+          capture-headers:
+            - X-Bar
+    db:
+      statement-sanitizer:
+        enabled: true
+  # Language-specific module options
+  java:
+    logback-appender:
```

Comment:
Should we use scope names here? Seems to make sense instead of introducing an alternative key.

Reply:
I think that's up to Java SIG. I've translated an example from https://github.com/MrAlias/otel-schema/issues/8#issuecomment-1291007692 but perhaps we just remove this example until it's decided how to represent Java's config?

Reply:
> Should we use scope names here? Seems to make sense instead of introducing an alternative key.

scope name is the "logger name" not the instrumentation library name

Reply:
Yeah that kind of throws a wrench in that idea. Side note: it would be great if we could record information about the name / version of the log appender instrumentation library in scope attributes. Right now, that information is erased.